server/lib/settings/migrator.ts

```diff
@@ -36,6 +59,10 @@ export const runMigrations = async (
         // something went wrong while saving file
         throw new Error('Unable to save settings after migration.');
       }
+    } else if (oldBackup) {
```

Comment:
We dont need this else if

Reply:
Yes we do? Why shouldn't we need this?

The point of this `else if` is to save the old backup. Because if no migration occurred the `settings.old.json` will then have the same content as the `settings.json` (because we saved current `settings.json` in `settings.old.json` it at the beginning). We don't want that, we want the `settings.old.json` to keep the value of the previous migration.

Reply:
Oh yeah I misunderstood. Ignore this comment